framework/platform/HLAPI/CMakeLists.txt

```
@@ -0,0 +1,43 @@
+project(hlapi VERSION ${prplmesh_VERSION})
```

maybe preceding with `prplmesh_`?
```suggestion
project(prplmesh_hlapi VERSION ${prplmesh_VERSION})
```